draft-release-notes.md

```diff
@@ -0,0 +1,354 @@
+    description: Staging environment for testing new cake recipes and features
+  - name: Local
+    url: http://localhost:3000/v1
+    description: Local development server running on your machine
+```
+
+#### Better polymorphic support
+
+Discriminator is a great way to match the correct schema to a payload. This release gives more robustness by adding support for a default type so that unknown objects can be handled safely.
+
+- The discriminator `propertyName` can now be an optional field.
+- New field `defaultMapping` to indicate which schema to use if the `propertyName` is not set, or if the value is unrecognized.
+- No change from previous versions: use `discriminator` to hint which entry in `anyOf` or `oneOf` is expected.
+- No change from previous versions: use `mapping` to link the discriminator property value to the Schema name if they aren't an exact match.
+- Implementations now SHOULD (rather than MAY) support templates/generics using `$dynamicRef`.
+
+Define the `mapping` and `discriminator` as before. The new field `defaultMapping` will be used if either the discriminator doesn't have a `propertyName` or when there is an object that doesn't match a `mapping` entry.
+
+```yaml
+  schemas:
+    Cake:
+      type: object
+      discriminator:
+        propertyName: cakeType
+        defaultMapping: sponge
+        mapping:
+          sponge: '#/components/schemas/SpongeCake'
+          chocolate: '#/components/schemas/ChocolateCake'
+          fruit: '#/components/schemas/FruitCake'
+      properties:
+        cakeType:
+          type: string
+          enum: [sponge, chocolate, fruit]
+        name:
+          type: string
+```
+
+#### Templates with formal syntax
+
+The specification now includes **ABNF** (Augmented Backus–Naur Form) for path templating, server variables, and runtime expressions in the Links object.
+This sounds terribly formal but it means that there is a clear syntax for what is supported in each of those locations, that existing tooling can understand.
+
+Next time you're wondering if you can do `/api/v{version}/users/{user-id}` in a path template, it'll be easy to check (spoiler: yes you can).
+
+#### Flexible response metadata fields
+
+- `description` field for responses are now optional (they used to be required but they could be empty).
+- Additional `summary` field for responses, useful when displaying responses in a list context.
+
+```yaml
+  /cakes:
+    get:
+      summary: List cakes
+      responses:
+        '200':
+          summary: Cake list
+          description: A list of cakes (this field is no longer required)
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+```
+
+#### Minor updates that are worth a mention
+
+- Non-Schema examples no longer "override" Schema examples; tools are free to use the most appropriate example for any given use case.
```

Comment:
Please let's not include this.  No one can remember what "override" meant, and given the widespread incorrect implementation of the thing usually doing the "override", this is hard to even explain.  Just let that language quietly disappear.  As best we can tell, it had no practical implications but was instead a somewhat philosophical statement about which examples are closest to the wire format.